gatsby-ssr.js

```diff
@@ -31,3 +32,21 @@ export const wrapRootElement = ({ element }) => (
 )
 
 export const wrapPageElement = ({ element }) => <Layout>{element}</Layout>
+
+export const onRenderBody = ({ setHeadComponents }) => {
+  if (storeConfig.analytics.gtmContainerId) {
+    setHeadComponents([
+      <GoogleTagManager
+        key="gtm"
+        containerId={storeConfig.analytics.gtmContainerId}
+        enablePartytown
+      />,
+      <Partytown key="party" />,
+    ])
+  } else if (process.env.NODE_ENV === 'development') {
+    // eslint-disable-next-line
+    console.warn(
+      'gtmContainerId is not set. Google Tag Manager will not work until you provide a valid container id.'
```

Comment:
This is already great! But how about also adding a link to https://developers.google.com/tag-platform/tag-manager/web#standard_web_page_installation, to make it easier for someone who doesn't know what the container ID is?

Reply:
Or even better! Since this is in the store's code, and users will get this from a starter, how about adding something like "Check the `analytics` section on your `store.config.js` file."